index.html

```diff
@@ -80,7 +85,7 @@ <h3>Tonic</h3>
         </div>
       </div>
       <div class="card">
-        <img src="assets/images/mobile/portfolio3.svg" alt="project1">
+        <div class="project3"></div>
         <div class="card-text">
           <div class="card-header">
             <h3>Tonic</h3>
```

Comment:
The `See Project` button is missing on the large laptop screen for some projects. Kindly see the screenshot below.
![missing2](https://user-images.githubusercontent.com/35504149/135631023-dab49eda-ac80-4f82-8463-9583ba0b35ce.png)